modules/op-installing-crunchy-postgres-database-and-tekton-hub.adoc

```diff
@@ -0,0 +1,165 @@
+= Optional: Installing Crunchy Postgres database and {tekton-hub}
+
+[role="_abstract"]
+Cluster administrators can install Crunchy Postgres database and configure {tekton-hub} to use it instead of the default database. 
+
+[discrete]
+.Prerequisites
+
+* Install the Crunchy Postgres Operator from the Operator Hub.
+* Create a Postgres instance that initiates a Crunchy Postgres database.
+
+[discrete]
+.Procedure
+
+. Get into the Crunchy Postgres pod.
+.Example: Get into the `test-instance1-m7hh-0` pod 
++
+[source,terminal]
+----
+$ oc exec -it -n openshift-operators test-instance1-m7hh-0 -- /bin/sh
+
+Defaulting container name to database.
+Use 'oc describe pod/test-instance1-m7hh-0 -n openshift-operators' to see all of the containers in this pod.
+sh-4.4$ psql -U postgres
+psql (14.4)
+Type "help" for help.
+----
+
+. Find the `pg_hba.conf` file.
++
+[source,terminal]
+----
+postgres=# SHOW hba_file;
+         hba_file         
+--------------------------
+ /pgdata/pg14/pg_hba.conf
+(1 row)
+
+postgres=#
+----
+
+. Exit from the database.
+
+. Check if the `pg_hba.conf` file has the entry `host all all 0.0.0.0/0 md5`, which is necessary for accessing all incoming connections. If necessary, add the entry at the end of the `pg_hba.conf` file.
++
+.Example: `pg_hba.conf` file
+[source,terminal]
+----
+sh-4.4$ cat /pgdata/pg14/pg_hba.conf
+
+# Do not edit this file manually!
+# It will be overwritten by Patroni!
+local all "postgres" peer
+hostssl replication "_crunchyrepl" all cert
+hostssl "postgres" "_crunchyrepl" all cert
+host all "_crunchyrepl" all reject
+hostssl all all all md5
+host  all  all 0.0.0.0/0 md5
+----
+
+. Save the `pg_hba.conf` file and reload the database.
++
+[source,terminal]
+----
+sh-4.4$ psql -U postgres
+psql (14.4)
+Type "help" for help.
+
+postgres=# SHOW hba_file;
+         hba_file         
+--------------------------
+ /pgdata/pg14/pg_hba.conf
+(1 row)
+
+postgres=# SELECT pg_reload_conf();
+ pg_reload_conf
+----------------
+ t
+(1 row)
+----
+
+. Exit the database.
+
+. Decode the secret value of the Crunchy Postgres host.
++
+.Example: Decode the secret value of a Crunchy Postgres host 
+[source,terminal]
+----
+$ echo 'aGlwcG8tcHJpbWFyeS5vcGVuc2hpZnQtb3BlcmF0b3JzLnN2YyA=' | base64 --decode
+test-primary.openshift-operators.svc
+----
+
+. Create a secret named `tekton-hub-db` in the target namespace with the following keys: 
+* `POSTGRES_HOST`
+* `POSTGRES_DB`
+* `POSTGRES_USER`
+* `POSTGRES_PASSWORD`
+* `POSTGRES_PORT`
++
+.Example: Custom database secrets
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: tekton-hub-db
+  labels:
+    app: tekton-hub-db
+type: Opaque
+stringData:
+  POSTGRES_HOST: test-primary.openshift-operators.svc 
+  POSTGRES_DB: test
+  POSTGRES_USER: test
+  POSTGRES_PASSWORD: woXOisU5>ocJiTF7y{{;1[Q( 
+  POSTGRES_PORT: '5432'
+...
+----
++
+[NOTE]
+====
+The default target namespace is `openshift-pipelines`.
+====
```

Comment:
Same comment as above. The Note section should be parallel to the example heading and the yaml box. In this case, parallel to the one under step 7, assuming that the previous example heading also is parallel to the one under step 7.